Vagrantfile

```diff
@@ -25,7 +25,7 @@ end
 # vagrant configurate
 Vagrant.configure(2) do |config|
   # select the box
-  config.vm.box = 'ubuntu/trusty64'
+  config.vm.box = 'bento/ubuntu-16.04'
```

Comment:
For example, in that moment ubuntu/xenial64 is broken: https://bugs.launchpad.net/cloud-images/+bug/1569237